pyanaconda/ui/gui/spokes/installation_source.glade

```diff
@@ -612,6 +612,23 @@
                                 <property name="position">2</property>
                               </packing>
                             </child>
+                            <child>
+                              <object class="GtkRadioButton" id="fromBootRadioButton">
```

Comment:
The radiobutton is missing left margin of 12, which is why the screenshot looks crappy.